packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -225,7 +258,7 @@ export default class SingleSelect extends React.Component<Props, State> {
                 keyboard={this.state.keyboard}
```

Comment:
I noticed that we're using `DropdownCore` here as well.  Maybe in a future PR we can update this to use `Dropdown` instead similar to how `ActionMenu` works.